README.md

```diff
@@ -3,17 +3,26 @@
 Bulk generation of images from Gosling visualization specifications
 
 ```bash
-node gosling-screenshot.js <input-dir> <output-dir> [format]
-# supported formats: png, jpeg, webp
+node gosling-screenshot.js
+# Usage: node gosling-screenshot.js [options] <input>
+#
+# Options:
+#   --format=[png|jpeg|webp] Output image format (default: png)
+#   --outdir=<directory>     Directory to save the output image
+#                             (required if input is a directory, optional otherwise)
+#
+# Arguments:
+#   input                    A Gosling specification file (JSON) or a directory of specifications
```

Comment:
@huyen-nguyen know you're crunched with time, but let me know if this new API is ok to merge!